tests/commands/remove.rb

```diff
@@ -56,7 +54,7 @@ def test_remove_essential_package
   def test_remove_package_with_essential_file
     assert_output(true) do
       until PackageUtils.installed?(@package_with_essential_file)
-        system "crew install -d #{@package_with_essential_file}", %i[out err] => File::NULL
+        system "yes | crew install -d #{@package_with_essential_file}", %i[out err] => File::NULL
```

Comment:
We can use `-f` now instead of `yes | ...`.

Reply:
Fixed, thanks.

Reply:
This causes hangs, so I've reverted it.